src/main/java/core/basesyntax/SqrtOperation.java

```diff
@@ -7,7 +7,10 @@
  * где вместо вопросика будет подставлено фактически переданное в метод число.</p>
  */
 public class SqrtOperation {
-    public double sqrt(double number) {
-        return 0.0;
+    public double sqrt(double number) throws MyException {
+        if (number < 0) {
+            throw new MyException("Expected non-negative number, got " + number);
```

Comment:
be more careful, in tass description it was mentioned to use exactly java.lang.IllegalArgumentException

Reply:
added java.lang.IllegalArgumentException